_notebooks/2020-03-27-covid19-progress-projections.ipynb

```diff
@@ -0,0 +1,2122 @@
+{
```

Comment:
<p>Why is this table Top 20 by one metric, but sorted by another metric?  This is confusing. Based on the title you are filtering this for the top 20 countries by immunity but then sorting by total cases?   The total number of new cases is not even shown in this table.</p><p> </p><p>What am I supposed to take way from this table?  Turkey will really have 80% immunity in 30 days?  Where is that coming from?  Doesn't quite look right to me</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/github/covid19-dashboard/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Added the sorting column and added clarifications.</p><p>Changed "Immunity" everywhere to "Affected percentage of population" to remove the speculative aspect of immunisation.</p><p> </p><p>The projection of Turkey heading to 80% in 30 days is based on the extremely high growth rate in recent days (above 30%). The model assumes constant infection rate. </p><p> </p><p>Although policy and fear will reduce the infection rate, I am not sure how we can model that reduction. </p><p> </p><p>Some counties have reacted quickly, some slowly, some have pre-empted the spread, and some are going exponential still. </p><p> </p><p>How would you propose to model the reduction in infection rate?</p><p> </p>